README.md

```diff
@@ -1,2 +1,79 @@
-# CESM GMOM_JRA configuration
-CESM GMOM_JRA configuration (MOM6-CICE6-DATM-DROF-SLND-SWAV-SGLC) to run with payu
+# MOM6-CICE6-datm-drof configurations for ACCESS-OM3
+
+This repository contains several
+[ACCESS-OM3](https://github.com/COSIMA/access-om3) configurations using the following components:
+
+- [MOM6](https://github.com/mom-ocean/MOM6)
```

Comment:
```suggestion
- [MOM6](https://github.com/NCAR/MOM6)
```

Reply:
Here the idea was to point to the "official" project repo, not the fork being used by OM3. Happy to change it if you think it is a possible source of confusion.

Reply:
I don't feel strongly at all, but I don't see the harm in pointing to the actual forks being used

Reply:
Or since this is a more user-oriented repo, should we link to the documentation rather than sources?

Reply:
Yes, documentation is actually better. What I really wanted was some sort of "official" project web page, but that does not seem to exist (except for the https://mom-ocean.github.io/ page, but that actually is more about MOM5 than MOM6).

I changed the MOM6, DATM and DROF links to point to the corresponding docs. Unfortunately I couldn't find any docs for CICE besides what's on their GitHub repo, so unless someone has a better idea,  I'll leave that one as it is.